supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -127,9 +127,9 @@
 [discrete]
 [[bimodal-it]]
 ==== image:images/caution.png[with caution] bimodal IT (noun)
-*Description*: "Bimodal IT" is the Gartner phrase for the combination of traditional (mode 1 or type 1) and modern (mode 2 or type 2) IT infrastructure and resources. There are many ways to talk about this combination approach. Using only the Gartner term can alienate other analysts or those not familiar with Gartner's phrasing.
+*Description*: _Bimodal IT_ is the Gartner phrase for the combination of traditional (mode 1 or type 1) and modern (mode 2 or type 2) IT infrastructure and resources. There are many ways to talk about this combination approach. Using only the Gartner term can alienate other analysts or those not familiar with Gartner's phrasing.
 
-The practice of having both modes together is often referred to as "hybrid", "agile", or "modern" IT. "Hybrid IT" is a more general term; for example, it could mean "on-premise plus public cloud". "Agile" and "modern IT" can both carry an implication of "mode 2". When using those terms, be specific about the exact technology combination you mean.
+The practice of having both modes together is often referred to as _hybrid_, _agile_, or _modern_ IT. _Hybrid IT_ is a more general term; for example, it could mean _on-premise plus public cloud_. _Agile_ and _modern IT_ can both carry an implication of _mode 2_. When using those terms, be specific about the exact technology combination you mean.
```

Comment:
This is another one that I'm thinking that quotes here was actually correct. But I'm not positive

Reply:
:heavy_check_mark: